src/highrl/utils/teacher_checker.py

```diff
@@ -199,23 +220,33 @@ def compute_difficulty(
     Returns:
         Tuple[float, int]: env area difficulty, env obstacles difficulty
     """
+    # Convert robot/goal position to integer for correct operations
```

Comment:
Why do positions have to be integers?

Reply:
Since the environment is store as a list, you need integer positions to access elements of this list. Hence, `floats` or continuous space wouldn't be feasible.

Reply:
Thank you